backend/VolontApp.API/Controllers/CoordinatorController.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using VolontApp.DAL.Repositories;
+using VolontApp.Models;
+
+namespace VolontApp.API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CoordinatorController : ControllerBase
+    {
+
+        private readonly CoordinatorRepository _CoordinatorRepository;
```

Comment:
Can be a simple private or public property, it does not matter. Also, watch out for the naming conventions! Use [ReSharper](https://www.jetbrains.com/resharper/).
Proposed changed : `public CoordinatorRepository CoordinatorRepository { get; set; }`